core-bundle/src/Resources/contao/modules/ModuleCloseAccount.php

```diff
@@ -91,6 +91,11 @@ protected function compile()
 			// Close account
 			if (!$objWidget->hasErrors())
 			{
+				if ($objWidget instanceof FinalizableWidgetInterface)
+				{
+					$objWidget->finalize();
+				}
+
```

Comment:
If we use `finalize()` here, don’t we need to use `$objWidget->validate(false);` on line 80 then too?

Same problem with other `finalize()` calls in this pull request (except `Form`). Doesn’t the widget now get finalized twice?

Reply:
True, I missed that.

Reply:
Should be fixed now.